airbyte-webapp/src/views/Connector/ServiceForm/components/Controls/ConnectorServiceTypeControl.tsx

```diff
@@ -178,15 +179,20 @@ const ConnectorServiceTypeControl: React.FC<{
   // This way, they will not be available for usage in new connections, but they will be available for users
   // already leveraging them.
   // TODO End hack
-  const disallowedOauthConnectors =
-    // I would prefer to use windowConfigProvider.cloud but that function is async
-    window.CLOUD === "true"
+  const workspace = useCurrentWorkspace();
+  const disallowedOauthConnectors = ([] as string[]).concat(
```

Comment:
@timroes I updated as per your comment. 
The only difference between this and the pattern you suggested is that this is flattening the array so we don't have nested arrays

Reply:
Thanks, if possible i'd prefer if we could stick to the more standard way of doing this in JS, that I mentioned in my previous comment for some reasons:

* Well it's kind of the standard, so I believe FE engineers will have an easier time reading it, that this construct using `concat` and spreading the parameters array into it.
* It doesn't require a TypeScript cast, which usually is better, since it doesn't leave us open for potential introducing bugs due to overwriting the type system later.
* This code is only living temporarily in here, and is planned to be removed in a couple of weeks again. Right now we're touching a lot of lines with this PR, which - once this change is removed - are basically just convuluted unused code, and need to be reverted to their prior state again. Using the spreading of arrays inside the array, would help us keeping this change way smaller, since it's just the one line added/removed (plus the `useCurrentWorkspace` above).

Reply:
my bad - I didn't parse your suggestion properly. fixed!